src/Raven.Server/Documents/Subscriptions/SubscriptionConnectionsState.cs

```diff
@@ -110,12 +110,20 @@ public HashSet<long> GetActiveBatches()
             return set;
         }
 
-        public Task AcknowledgeBatch(SubscriptionConnection connection, long batchId, List<DocumentRecord> addDocumentsToResend)
+        public async Task AcknowledgeShardingBatchAsync(string changeVector, string cvForOrchestrator, long batchId, List<DocumentRecord> addDocumentsToResend)
```

Comment:
this should not be in this class?

Reply:
shouldn't you just override the 'AcknowledgeBatchAsync' in one of the classes that inherit from this?

Reply:
I had to pass another parameter to the method: `string cvForOrchestrator` so I cannot simply override `AcknowledgeBatchAsync`